### PR TITLE
Tweak rpm generation

### LINKED ISSUE #4378: Fix the specfiles and RPMs since the split of packages
We are now building RPMs automatically with each PR, but the layout of the RPMs is slightly different to what we had before.

Previously, we had one package (srpm) that produced multiple sub-packages,that is:

* bodhi-client-5.7.4-0.fc34.noarch.rpm
* bodhi-composer-5.7.4-0.fc34.noarch.rpm
* bodhi-docs-5.7.4-0.fc34.noarch.rpm
* bodhi-server-5.7.4-0.fc34.noarch.rpm
* python3-bodhi-5.7.4-0.fc34.noarch.rpm
* python3-bodhi-client-5.7.4-0.fc34.noarch.rpm
* python3-bodhi-messages-5.7.4-0.fc34.noarch.rpm

the way it is set up now (probably incorrectly because i did it) is 3 srpms / specfiles producing each module as a package: (bodhi-client, bodhi-server, and bodhi-messages)

We need to decide if we still want to split out composer and do that, and also the python3* packages too

### PR DESCRIPTION
**Pull request by ryanlerch**
Slightly change how we make the rpms. First is to change the way we
name them based on the suggestions in #4378 

THe second is to add a time to the end of the date in the version
for the snapshot versioning so have more flexibility.

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

File: bodhi-messages/bodhi-messages.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 %global pypi_name bodhi-messages
 %global pypi_version 5.7.4
 
-Name:           %{pypi_name}
+Name:           python3-%{pypi_name}
```
Comment (nphilipp) [inline bodhi-messages/bodhi-messages.spec]: This doesn't conform with the [Fedora Python Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_library_naming): for a package containing primarily a Python library, the source package (aka the "component") must be named `python-…`, and the built package must be named `python3-…`. This means we need a new `%package python3-%{pypi_name}` section.

**Review by nphilipp — CHANGES_REQUESTED** [review]
Other than the source package naming thing, LGTM!

File: bodhi-messages/bodhi-messages.spec (hunk 1 above)
Comment (ryanlerch) [reply bodhi-messages/bodhi-messages.spec]: I added this section -- i think it is correct :)

**Review by nphilipp — APPROVED** [review]
LGTM! 👍

**Pull request merged**